pkg/dbutil/table_test.go

```diff
@@ -33,6 +34,7 @@ type testCase struct {
 	sql     string
 	columns []string
 	indexs  []string
+	colLen  [][]int
```

Comment:
not this type of UT, I mean we should check the https://github.com/pingcap/tidb-tools/issues/590 can has expected SQL